frontend/src/pages/home/Home.tsx

```diff
@@ -0,0 +1,72 @@
+import * as React from 'react';
+import { Button, Hint, HintBody, HintTitle, PageSection } from '@patternfly/react-core';
+import { TimesIcon } from '@patternfly/react-icons';
+import { useNavigate } from 'react-router-dom';
+import { useLocalStorage } from '@patternfly/quickstarts';
+import ProjectsSection from '~/pages/home/ProjectsSection';
+import OrganizeSection from '~/pages/home/OrganizeSection';
+import QuickStartsSection from '~/pages/home/QuickStartsSection';
+import EnableTeamSection from '~/pages/home/EnableTeamSection';
+import { AccessReviewResourceAttributes } from '~/k8sTypes';
+import { useAccessReview } from '~/api';
+
+const accessReviewResource: AccessReviewResourceAttributes = {
+  group: 'project.openshift.io',
+  resource: 'projectrequests',
+  verb: 'create',
+};
+
+const Home: React.FC = () => {
+  const navigate = useNavigate();
+  const [hintHidden, setHintHidden] = useLocalStorage('rhodsNewLandingPageMessage', null);
+  const [allowCreate, rbacLoaded] = useAccessReview(accessReviewResource);
```

Comment:
Beyond Self Provisioner -- we have the ability to disable Projects

![image](https://github.com/opendatahub-io/odh-dashboard/assets/8126518/47c095bf-3224-4e2c-9f90-29ab27f82b70)

... but cards:

![Screenshot 2024-03-25 at 2 30 29 PM](https://github.com/opendatahub-io/odh-dashboard/assets/8126518/8cd09d05-aa8f-4f44-ba70-71cd1988a06a)

... also... shouldn't I see my projects that DO have access to when I'm on this page? Or am I mis-remembering the "Day 1" review and what we intended to provide on v1?

![image](https://github.com/opendatahub-io/odh-dashboard/assets/8126518/6cc4a2c6-3473-4971-bcf4-32f9384396a4)

Reply:
I added the check.  For this iteration we are not showing existing projects. That will be done in a follow-up.